.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
> it's more an application than a library

Is there a reason for this thought?

I would ignore the `composer.lock` too, assuming it is indeed a library (as stated in the `composer.json`) and considering we don't need to lock deps at point releases